workflows/prognostic_c48_run/runtime/segmented_run/append.py

```diff
@@ -30,6 +32,16 @@ def read_run_config(run_url):
     return fv3config.load(io.BytesIO(s))
 
 
+def print_step_metadata_json(run_url: str, config: dict):
+    info = StepMetadata(
+        job_type="prognostic_run",
+        commit=os.environ.get("COMMIT_SHA"),
+        url=run_url,
+        dependencies={"ml_models": get_model_urls(config)},
```

Comment:
I think this violates the `Mapping[str,str]` usage, but prephysics would have a list of different models anyways?  Any way to expand it out so its a {"<unique_model_name>": "url"}  Or maybe it makes sense to relax the restrictions.  What matters is that these are useful in a query.

Reply:
I don't think so, since there's no key data in the user config, it just provides models in a list. I can separate prephysics models and the rest of the models out into individual keys.

Reply:
On second thought I think I will leave all the ML models in a single list. For the purposes of tracking dependencies I don't think there should be multiple keys to search on.